pkg/agent/controller/controller_suite_test.go

```diff
@@ -280,10 +280,22 @@ func (c *cluster) start(t *testDriver, syncerConfig broker.SyncerConfig) {
 
 	serviceExportCounterName := "submariner_service_export" + bigint.String()
 
+	bigint, err = rand.Int(rand.Reader, big.NewInt(1000000))
+	Expect(err).To(Succeed())
```

Comment:
is there a gomega `To(BeNil())` match?
I find the statement "expect error to succeed" an oxymoron, but maybe it's just me